OrbitClientServices/ProcessClient.cpp

```diff
@@ -95,6 +99,7 @@ ErrorMessageOr<std::string> ProcessClient::FindDebugInfoFile(const std::string&
 
 ErrorMessageOr<std::string> ProcessClient::LoadProcessMemory(int32_t pid, uint64_t address,
                                                              uint64_t size) {
+  ORBIT_SCOPE_FUNCTION;
```

Comment:
Question - does __FUNCTION__ include class name?

Reply:
With Clang it doesn't. With msvc it does, as well as the namespaces.